pkg/utils/gardener/shoot.go

```diff
@@ -788,6 +788,13 @@ func DefaultGVKsForEncryption() []schema.GroupVersionKind {
 	}
 }
 
+// DefaultGRsForEncryption returns the list of GroupResources which are encrypted by default.
+func DefaultGRsForEncryption() []schema.GroupResource {
```

Comment:
I do not think that shortening the name makes sense here.
```suggestion
func DefaultGroupResourcesForEncryption() []schema.GroupResource {
```